WF-PCInstaller/plugin_dir/utils/apache_utils.py

```diff
@@ -0,0 +1,221 @@
+import re
```

Comment:
I might call this `/plugin_dir/common/apache.py` -- same convention for the other common things